ui/apps/platform/src/Components/CompoundSearchFilter/components/SearchFilterAutocomplete.tsx

```diff
@@ -96,11 +102,32 @@ function SearchFilterAutocomplete({
         []
     );
 
+    const autocompleteSearchString = `${searchTerm}:${filterValue ? `r/${filterValue}` : ''}`;
+
+    const searchContext = {
+        ...searchFilter,
+        ...additionalContextFilter,
+    };
+    const filteredSearchContext = Object.keys(searchContext).reduce((acc, key) => {
+        // Autocomplete requests for some filters never return results if there is a 'Fixable' search filter
+        // included in the query.
+        if (key !== 'FIXABLE') {
```

Comment:
Also @dvail, I looked at the special case you mentioned [here](https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/components/FilterAutocomplete.tsx#L63-L93). It looks like the Fixable filter is the only issue though. I added the search filters for CVE and Severity, and the autocomplete options showed up like normal. If I just remove "FIXABLE" from the search context, everything works like normal.

Reply:
Ok good to know. Possibly something to discuss with the BE beyond 4.5 but having context for all other search filters is still very beneficial.